cinq_collector_aws/account.py

```diff
@@ -128,12 +109,11 @@ def update_s3buckets(self):
 
                 except Exception as e:
                     self.log.error('Could not retrieve bucket statistics / {}'.format(e))
+                    metrics = {}
```

Comment:
Should we add an error message here that informs us that the metrics weren't found?

```suggestion
                    metrics = {"found": false}
```

Reply:
I think this is a good case to use unavailable